test/powershell/Modules/Microsoft.PowerShell.Core/RemotingCmdlets.Tests.ps1

```diff
@@ -13,7 +13,17 @@ Describe "SSH Remoting Cmdlet Tests" -Tags "Feature" {
     }
 
     It "Invoke-Command HostName parameter set should throw error for invalid key path" {
-        { Invoke-Command -HostName localhost -UserName User -KeyFilePath NoKeyFile -ScriptBlock {1} } |
+        { Invoke-Command -HostName localhost -UserName User -KeyFilePath NoKeyFile -ScriptBlock { 1 } } |
             Should -Throw -ErrorId "PathNotFound,Microsoft.PowerShell.Commands.InvokeCommandCommand"
     }
+
+    It "Invoke-Command should support positional parameter ScriptBlock when using HostName" {
+        { Invoke-Command -HostName localhost { "test" } } |
```

Comment:
Instead of checking it in this way, i would prefer checking it using command metadata. I have a sample here:

```powershell
Describe "sdfsf" {

    It "Invoke-Command should support positional parameter ScriptBlock when using parameter set '<ParameterSetName>'" -TestCases @{ParameterSetName = 'SSHHost'}, @{ParameterSetName = 'SSHHostHashParam'} {
        param ($ParameterSetName)
        $commandInfo = Get-Command -Name Invoke-Command 

        $sshHostParameters = $commandInfo.ParameterSets | Where-Object { $_.Name -eq $ParameterSetName } 
        
        $sshHostSBPostion = $sshHostParameters.Parameters | Where-Object { $_.name -eq 'ScriptBlock' } | Select-Object -ExpandProperty position
        
        $sshHostSBPostion | Should -Be 1        
    }
}
```